config/locales/en.yml

```diff
@@ -401,6 +401,8 @@ en:
     not_found: Page not found.
     trashed_vacancy_found: Page not found.
     server_error: Something went wrong at our end.
+    external_server_error:
+      dfe_sign_in: There was a problem with DfE Sign-in, please try again in a moment.
```

Comment:
Do we know if Michael has reviewed this line?

Reply:
I'm not sure, but I don't think he has

Reply:
The plan is to get rid of this custom page.  See: https://trello.com/c/fR9rlofE

Reply:
In that case I will leave this as is and then remove it in a future commit as part of the work on the linked Trello card